packages/elements-core/src/components/Docs/HttpService/AdditionalInfo.tsx

```diff
@@ -0,0 +1,45 @@
+import { Panel } from '@stoplight/mosaic';
+import { IHttpService } from '@stoplight/types';
+import React from 'react';
+
+import { MarkdownViewer } from '../../MarkdownViewer';
+
+const spdxLicenseList = require('spdx-license-list');
```

Comment:
Did you try `import` instead of `require`? Just seems odd to have a `require` in here...

Reply:
addressed with abc8aac